indexing-hadoop/src/main/java/io/druid/indexer/hadoop/DatasourceIngestionSpec.java

```diff
@@ -48,7 +51,15 @@ public DatasourceIngestionSpec(
   )
   {
     this.dataSource = Preconditions.checkNotNull(dataSource, "null dataSource");
-    this.interval = Preconditions.checkNotNull(interval, "null interval");
+
+    List<Interval> theIntervals = null;
+    if (interval != null) {
+      theIntervals = ImmutableList.of(interval);
+    } else if (intervals != null && intervals.size() > 0) {
+      theIntervals = JodaUtils.condenseIntervals(intervals);
+    }
```

Comment:
If we do end up supporting both, probably makes sense to fail if both are provided.